EventsExpress/Controllers/EventController.cs

```diff
@@ -27,13 +30,15 @@ public class EventController : ControllerBase
         private readonly IEventService _eventService;
         private readonly IMapper _mapper;
         private readonly ISecurityContext _securityContextService;
+        private readonly IValidator<EventViewModel> _validator;
 
-        public EventController(IEventService eventService, IMapper mapper, ISecurityContext securityContextService, IPhotoService photoService)
+        public EventController(IValidator<EventViewModel> validator, IEventService eventService, IMapper mapper, ISecurityContext securityContextService, IPhotoService photoService)
```

Comment:
Remove this validator from the constructor if you are injecting into methods then.

Reply:
done